### PR TITLE
make days more respomsive for mobile

### LINKED ISSUE #1: Make website look better on mobile
Needs better responsive design

### PR DESCRIPTION
**Pull request by butlerx**
close #1

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -97,3 +98,7 @@ ul {
 #livestream {
 	display: none;
 }
+
+.countainer {
```
Comment (SeanHealy33) [inline css/main.css]: `countainer` -> `container`

File: css/main.css [hunk 2]
```diff
@@ -38,6 +38,7 @@ video {
 .day {
 	margin-top: 2vh;
 	margin-bottom: 5vh;
+  padding: 0;
```
Comment (SeanHealy33) [inline css/main.css]: Inconsistent indentation

**Comment by SeanHealy33** [comment]
not really sure how these changes make the site responsive

**Comment by butlerx** [comment]
Too much padding made stuff too small on mobile with lots of white space

**Comment by butlerx** [comment]
this pull request is having some weird merge conlict on files goign to close an reopen

File: dist/css/main.min.css [hunk 3]
```diff
@@ -1 +1 @@
...
...0;src:local('Roboto Light'),local('Roboto-Light'),url(http://fonts.gstatic.com/s/roboto/v15/Hgo13k-tfSpn0qi1SFdUfaCWcynf_cDxXwCLxiixG1c.ttf) format('truetype')}@font-face{font-family:Roboto;font-style:normal;font-weight:400;src:local('Roboto'),local('Roboto-Regular'),url(http://fonts.gstatic.com/s/roboto/v15/zN7GBFwfMP4uA6AR0HCoLQ.ttf) format('truetype')}header h1{margin:.5em 0 .3em;font-size:15vmin;font-weight:100}header h1 span,p{font-weight:300}header h1 span{color:red}header ul{list-style-type:none;margin:0;padding:0;overflow:hidden;display:flex;flex-wrap:wrap;justify-content:center}header ul li{float:left;padding:0 8px}video{width:100%}.day{margin-top:2vh;margin-bottom:5vh;padding:0}.day-name{width:auto;display:inline}.event,.place,.time{display:inline-block}.time{float:left;font-size:1.5vh;width:15%;max-width:5em}.event{width:60%}.place{color:#000;float:right}ul{margin-bottom:2vh}.collapsable-header{height:initial;-webkit-transition:all .2s}#countdown{display:none}#countdown .counter{display:flex;flex-wrap:wrap;flex-direction:column;font-size:medium}#countdown .counter span{display:block;font-size:2em}#livestream{display:none}#playlist{display:block}.container{padding:0 1rem}
```
Comment (VoyTechnology) [inline dist/css/main.min.css]: We should not be including the autogenerated files in the repo

Comment (VoyTechnology) [inline dist/css/main.min.css]: maybe add `*.min.js` to `.gitignore`

Comment (butlerx) [inline dist/css/main.min.css]: i will do this once i set up a postinstall to generate them for deployment

Comment (VoyTechnology) [inline dist/css/main.min.css]: But then you will have to edit the history to remove this file

Comment (butlerx) [inline dist/css/main.min.css]: well ive done it

**Pull request merged**